cms/djangoapps/contentstore/management/commands/tests/test_migrate_to_split.py

```diff
@@ -107,6 +108,17 @@ def test_locator_string(self):
             "org.dept", "name", "run",
         )
         split_store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.split)
-        locator = split_store.make_course_key(self.course.id.org, self.course.id.course, self.course.id.run)
-        course_from_split = modulestore().get_course(locator)
+        locator = split_store.make_course_key("org.dept", "name", "run")
```

Comment:
we are giving the new course_id to migrate_to_split  command at line no **108** and now using this new course_id arguments to build split course_id.